### ISSUE TITLE
Problem connecting to heroku

### ISSUE THREAD

**Issue by meatballs** [Borvik/vscode-postgres#54]
Possibly related to #32 

When I try to connect to a db hosted at heroku, I get the  "no pg_hba.conf entry for host ...." error both in the db explorer and when I try to run a script.

I have other clients (PGAdmin and Oracle SQL Developer) on my laptop (Mac) which connect ok and I can execute queries from them.

**Comment by Borvik** [comment]
I have just attempted a connection to a Heroku database, successfully.

Here are the connection details I used - following the database settings page on Heroku:
```
{
  "label": "Heroku Test",
  "host": ".........compute-1.amazonaws.com",
  "user": ".........",
  "port": 5432,
  "certPath": "/home/...../image012.png",
  "database": ".......",
  "password": "......."
}
```
Database must be set - it can't enumerate them.
Also currently certPath must be set (can be something other than an actual certificate) - this triggers SSL mode, which is required for Heroku.
